postgres/datadog_checks/postgres/postgres.py

```diff
@@ -533,6 +537,7 @@ def _connect(self, host, port, user, password, dbname, ssl):
                     database=dbname,
                     sslmode=ssl,
                     application_name="datadog-agent",
+                    options='-c statement_timeout=%s' % query_timeout if query_timeout else None,
```

Comment:
You should refactor to not have to repeat the same statement 3 times.

Reply:
doing that in a separate PR